src/doc/src/guide/cargo-toml-vs-cargo-lock.md

```diff
@@ -16,8 +16,8 @@ We recommend pairing this with
 
 Let’s dig in a little bit more.
 
-`Cargo.toml` is a [**manifest**][def-manifest] file in which we can specify a
-bunch of different metadata about our package. For example, we can say that we
+`Cargo.toml` is a [**manifest**][def-manifest] file in which you can specify a
+bunch of different metadata about our package. For example, you can say that you
```

Comment:
Should we split pronoun switch into a separate PR? The purpose is quite different from using simple English and people may have different preference on it?

Reply:
Sure - separated the commits so will be easy for me to split

Reply:
Reverted the pronoun component

Reply:
Thanks!